_settings/settings-local.py

```diff
@@ -22,34 +22,50 @@
     'django_user_agents.middleware.UserAgentMiddleware',
 )
 
-DEBUG = True
-USE_TZ = True
-
 TIME_ZONE = 'America/Vancouver'
 
-################ LOCAL MYSQL ##################
-# DATABASES = {
-#     'default': {
-#         'ENGINE': 'django.db.backends.mysql',
-#         'NAME': 'ubyssey',
-#         'USER': 'root',
-#         'PASSWORD': 'ubyssey',
-#         'HOST': '127.0.0.1',
-#         'PORT': '3306',
-#     },
-# }
+DEBUG = True
+USE_TZ = True
 
+DEV_OPTIONS = ['LOCAL_DOCKER', 'LOCAL', 'PRODUCTION']
+DEV_ENV = DEV_OPTIONS[0]
+
+###############  PRD MYSQL  #################
+if DEV_ENV == DEV_OPTIONS[2]:
```

Comment:
Don't think defining prd config in local file since this file is [not being loaded](https://github.com/ubyssey/ubyssey.ca/blob/develop/.travis.yml#L9) by travis

Reply:
This is for PRD database connection, ie when you want to proxy into the database